Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,45 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: View
+	var guess: String
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		guess = ""
```

Comment:
My solution was to create a `GuessManager` method called `assignGuess(view)` which passes in the `view` instance from `Game` and calls `view.receiveGuess`. When I needed to assign or reassign guess from a `Game` function, I did a `guessManager.guess = guessManager.assignGuess`. Any `Game` method that passed in the guess now is not, and the state of the guess is removed from both the `Game` and `View`